### DISCUSSION TITLE
There is no max-w-screen-* class in default conf

### DISCUSSION THREAD

**Discussion by Jerry-gif** [tailwindlabs/tailwindcss#3546] [Help] [answered]
Hello!

I couldn't find a class max-w-screen-2xl in my main css file. What am I doing wrong?

Importing:
@tailwind base;
@tailwind components;
@tailwind utilities;

**Comment by simonswiss** [comment, marked as answer]
Hey!

Are you sure the file where you use this `max-w-screen-2xl` class is added in the array of paths in your your `purge` config?

What's the rest of your Tailwind config file looking like? Any chance you disabled the `2xl` `screen`?

Are you able to reproduce in a Tailwind Play example?

**Reply by Jerry-gif** [reply]
You were right, there is mistype in my tailwind.config.js. I didn't look into. Thank you!
